mlflow/server/__init__.py

```diff
@@ -29,6 +26,10 @@ def _add_static_prefix(route):
     return route
 
 
+for http_path, handler, methods in handlers.get_endpoints():
```

Comment:
@xhochy I think this change actually makes more conceptual sense than the one I proposed in https://github.com/mlflow/mlflow/pull/1413#pullrequestreview-249034945 (prefixing only `ajax-api` endpoints with the static prefix), but unfortunately I think it's a breaking change if we make the `--static-prefix` change the standard REST API endpoints in addition to the ajax-api endpoints. Thus maybe we can stick with modifying [this  line](https://github.com/mlflow/mlflow/blob/fcb6fa8d18c3f8981806f220f65c3888f99a8046/mlflow/server/handlers.py#L373) to add the static prefix only to the ajax-api endpoints. Otherwise the PR LGTM!

Reply:
Do you mean that I should add the following to also support the old API paths?
```suggestion
for http_path, handler, methods in handlers.get_endpoints():
    app.add_url_rule(http_path, handler.__name__, handler, methods=methods)
```

Reply:
I was actually thinking we'd change `_get_paths` to be:

```
def _get_paths(base_path):
    """
    A service endpoints base path is typically something like /preview/mlflow/experiment.
    We should register paths like /api/2.0/preview/mlflow/experiment and
    /ajax-api/2.0/preview/mlflow/experiment in the Flask router.
    """
    return ['/api/2.0{}'.format(base_path), _add_static_prefix('/ajax-api/2.0{}'.format(base_path))]
```

The only difference from your proposed change is that we only serve the REST API from `/api/2.0/...` rather than serving from both `/api/2.0/...` and `<static-prefix>/api/2.0/...`